spec/features/publication_workflow_spec.rb

```diff
@@ -10,6 +10,7 @@
   let(:work) { FactoryGirl.create(:image) }
   context 'a logged in user' do
     before do
+      Image.delete_all
```

Comment:
What do you think about using the `:clean` tag here instead: `RSpec.feature '...', :clean do; ...; end`?